tests/benchmarks/test_xarray.py

```diff
@@ -0,0 +1,70 @@
+import uuid
+
+import fsspec
+import pytest
+from coiled import Cluster
+from distributed import Client
+from xarray.groupers import TimeResampler
+
+from tests.conftest import dump_cluster_kwargs
+from tests.utils_test import wait
+
+xr = pytest.importorskip("xarray")
```

Comment:
From what it looks like, we can skip the `importorskip` statements, otherwise this shouldn't have worked in the first place. 

IIUC, both `xarray` and `flox` are installed via `ci/environment.yml` which is the basis for all CI jobs and the development environment. If I'm missing something, the suggestion below should fix the order of skips and imports.

```suggestion
from tests.conftest import dump_cluster_kwargs
from tests.utils_test import wait

xr = pytest.importorskip("xarray")
from xarray.groupers import TimeResampler
```

Reply:
good point.

the skips are for AB tests if you run on smaller environments